### ISSUE TITLE
Shortcuts on the field's menu

### ISSUE THREAD

**Issue by tleb** [MaKleSoft/padlock#39]
I have started using padlock and I love it! I would just love to be able to keep my hands on my keyboard when I use it. In this aspect, I think it would be nice if in the field's menu, every option would have a shortcut: `c` for "Copy", `e` for "Edit", `g` for "Generate" and `r` for "Remove".
It could be done on every menu.

Also, above the records' list, it says "tap to search...". I tried typing but it did nothing. I understood we need to click on the text. It would be nice to focus the input when we start typing.

I looked at the source code for both of those but I couldn't get my head around it. I'm not really a front-end type of guy, sorry. :smile:

**Comment by MaKleSoft** [comment]
Great suggestions, thanks! Shortcuts for the field menu are definitely on the todo-list. As for the search: Have you tried using `ctrl/cmd + F`?

**Comment by tleb** [comment]
I didn't when I wrote that. Now, I have. :)

Still, as nothing is binded on the keys (`[a-z]`), it could be nice to just bind the search to it and add the typed letter at the start, so that no Ctrl+F nor click is needed to search. A bit like on a Windows menu, when we start typing, even if the input isn't selected, it searches.

Also, I figured out to add the app to our path, if you want, I can do a PR to explain how, for those people who don't like the app menu of Chrome.
